recipe/0001-remove-superfluous-build-requirement.patch

```diff
@@ -0,0 +1,25 @@
+From 44596274a63887b0c17c4ae39c24ce39ad1d8a5c Mon Sep 17 00:00:00 2001
+From: Mohamed Sentissi <msentissi@anaconda.com>
+Date: Wed, 18 Sep 2024 14:37:32 -0400
+Subject: [PATCH] remove superfluous build requirement
+
+---
+ setup.py | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/setup.py b/setup.py
+index df9f32f..ff83522 100644
+--- a/setup.py
++++ b/setup.py
+@@ -15,7 +15,7 @@
+ from setuptools import setup
+ 
+ 
+-requirements = ["requests>=2.20.0,<3.0"]
++requirements = []
```

Comment:
thanks for the patch!

unfortunately, this will empty also `install_requires`, see https://github.com/googlemaps/google-maps-services-python/blob/v4.10.0/setup.py#L38-L39

you could override just `setup_requires` with:
```
    setup_requires=[],
    install_requires=requirements,
```

Reply:
I did miss that it applied to both! Fixed it in the latest commit.